repository/n.json

```diff
@@ -297,6 +297,16 @@
 			]
 		},
 		{
+			"name": "Network Tech",
+			"details": "https://github.com/heyglen/network_tech",
+			"releases": [
+				{
+					"sublime_text": ">=3000",
```

Comment:
This needs to be changed to `"sublime_text": ">=3092",` since that is the first release that has solid support for `.sublime-syntax`.